pom.xml

```diff
@@ -195,28 +195,75 @@
                 <repository>
                     <id>evolveum-nexus-releases</id>
                     <name>Internal Releases</name>
-                    <url>http://nexus.evolveum.com/nexus/content/repositories/releases/</url>
+                    <url>https://nexus.evolveum.com/nexus/content/repositories/releases/</url>
                 </repository>
                 <repository>
                     <id>evolveum-nexus-snapshots</id>
                     <name>Internal Releases</name>
-                    <url>http://nexus.evolveum.com/nexus/content/repositories/snapshots/</url>
-                </repository>
-                <repository>
-                    <id>apache-snapshots</id>
-                    <name>Apache Snapshots</name>
-                    <url>http://repository.apache.org/snapshots/</url>
+                    <url>https://nexus.evolveum.com/nexus/content/repositories/snapshots/</url>
                 </repository>
             </repositories>
             <dependencies>
                 <dependency>
                     <groupId>com.evolveum.midpoint.gui</groupId>
                     <artifactId>admin-gui</artifactId>
-                    <version>4.0.2</version>
+                    <version>4.4.4</version>
                     <type>jar</type>
                     <classifier>classes</classifier>
                     <scope>provided</scope>
                 </dependency>
+                <dependency>
```

Comment:
What are the `testng` and `connector-framework-contract` dependencies needed for?  Just want to make sure if these are needed and should be commited?

Reply:
@mike-elabs

Dependencies had to be adjusted and added to make midPoint work in the Integration Test with midPoint 4.4. However, there is currently no midPoint and no automated Integration Test. I will therefore add a commit to remove this profile itself.